resources/wolframPackage.wl

```diff
@@ -0,0 +1,20 @@
+(* ::Package:: *)
+
+  BeginPackage[ "WolframPackage`"]
```

Comment:
Perhaps make it very explicit the name is arbitrary?
```suggestion
  BeginPackage[ "DemoWolframPackage`"]
```

Reply:
Good thought.